emba.sh

```diff
@@ -49,6 +50,7 @@ import_module()
 
 main()
 {
+  INVOCATION_PATH="$(dirname "$0")"
```

Comment:
1st - cosmetic updates like this (we can accept this)

Reply:
This is not only cosmetic: I needed that for this feature.